docs/storage.md

```diff
@@ -461,40 +461,66 @@ For the full discussion about Ceph in XCP-ng, see this forum thread: <https://xc
+- Local (not recommended): Local ISO SR is a directory created directly on the dom0. It's only accessible on the pool where the dom0 ith the directory on it.
 
-In [Xen Orchestra](management.md#xen-orchestra) go into "New/Storage" and select "ISO SR":
+Create a Shared ISO SR:
 
-![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+First, you need to create the NFS or SMB Share. The creation depends on the device you want to use to store your ISO, but here a small tutorial to mount a NFS share on Debian 11:
 
-Usually, you can use a NFS or SMB share. But if you don't have any of those, you can rely on a local ISO SR that you create yourself:
+```
+fdisk /dev/xvdb #create a new partition
+mkfs.ext4 -b 4096 /dev/xvdb1 # format the partition to ext4
+apt install nfs-kernel-server
+mkdir /ISOs
+mount /dev/xvdb1 /ISOs #mount the disks to /ISOs
+emacs /etc/fstab
+  /dev/xvdb	  /ISOs	  /ext4	  defaults	0	1
+emacs /etc/exports
+  /ISOs 192.168.X.0/24(rw,sync,no_subtree_check,no_root_squash)
+exportfs -arv
+```
 
-1. Select "Local" instead of NFS/SMB
-2. Enter a path that already exists on your physical host
-3. Upload ISO on your host to the same path
+Then, in Xen Orchestra go into "New/Storage" and select "ISO SR":
 
-That's it!
+![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+
+Create a Local ISO SR:
 
 From the CLI:
 
-1. Create a directory on the local filesystem to storage your ISOs
-2. Copy/move ISOs to this new location
-3. Create the ISO SR using xe sr-create
-4. You can add or update ISOs later by placing them into the directory you created in step 1
-5. Rescan the SR if you change the files stored in the ISO directory
+    Create a directory on the local filesystem to storage your ISOs
+    Copy/move ISOs to this new location
+    Create the ISO SR using xe sr-create
+    You can add or update ISOs later by placing them into the directory you created in step 1
+    Rescan the SR if you change the files stored in the ISO directory
 
 Here's an example creating a Shared ISO SR named "ISO Repository" that will be stored in /opt/var/iso_repository:
+
 ```
 mkdir -p /opt/var/iso_repository
 
 xe sr-create name-label="ISO Repository" type=iso device-config:location=/opt/var/iso_repository device-config:legacy_mode=true content-type=iso
-a6732eb5-9129-27a7-5e4a-8784ac45df27
+a6732eb5-9129-27a7-5e4a-8784ac45df27 # this is the output
 
 xe sr-scan uuid=a6732eb5-9129-27a7-5e4a-8784ac45df27
 ```
 If your host is in a pool of several hosts, you need to add the `host-uuid` parameter to the `xe sr-create` command above. You can retrieve the host UUID with `xe host-list`.
+
+You can then upload your ISO in /opt/var/iso_repository/
+
+On Xen Orchestra, go into "New/Storage" and select "ISO SR"
+
+    Select "Local" instead of NFS/SMB
+    Enter the path created before
+    Upload ISO on your host to the same path
```

Comment:
This will look like a code block.

I think a bullet list fits better for this.

```suggestion
* Select "Local" instead of NFS/SMB
* Enter the path created before
* Upload ISO on your host to the same path
```